qa/standalone/misc/ver-health.sh

```diff
@@ -60,7 +59,7 @@ function TEST_check_version_health_1() {
 
     kill_daemons $dir KILL osd.1
     EXTRA_OPTS=" --debug_version_for_testing=01.00.00-gversion-test" activate_osd $dir 1
-    sleep 5
+    sleep 10
```

Comment:
is there a more reliable way to check for version change instead of sleep to make the test more robust?

Reply:
Instead of every sleep 10 we could loop 10 times with sleep 1 until we see the state change.  It would make the test run faster, but the test more complicated.

Reply:
I think waiting until state change is more reliable.